sdk/storage/azure-storage-blob-cryptography/README.md

```diff
@@ -21,7 +21,7 @@ This package supports client side encryption for blob storage.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-blob-cryptography</artifactId>
-  <version>12.10.0</version>
+  <version>12.11.0-beta.1</version>
```

Comment:
we generally, keep README at stable versions, so existing and new customers try the stable release.